CONTRIBUTING.md

```diff
@@ -35,7 +35,7 @@ npm test
 
 ## Pull Requests
 ### Technical Requirements
-1. Please lint all proposed changes with the `npm lint-fix` command
+1. Please lint all proposed changes with the `npm run lint --fix` command
```

Comment:
there is no `npm lint-fix` command, there is a `lint` action in package.json scripts... I think it means this?

Reply:
That command was removed by a recent PR if I recall correctly, I believe it's `npm run format` now